django/contrib/staticfiles/storage.py

```diff
@@ -220,10 +220,14 @@ def converter(matchobj):
             matched = matches["matched"]
             url = matches["url"]
 
-            # Ignore absolute/protocol-relative and data-uri URLs.
+            # Ignore URLs that include a protocol
             if re.match(r"^[a-z]+:", url):
                 return matched
 
+            # Ignore protocol-relative URLs
+            if url.startswith("//"):
+                return matched
```

Comment:
I would add it to the previous check, e.g.
```python
# Ignore absolute/protocol-relative and data-uri URLs.
if re.match(r"^([a-z]+:|//)", url):
```

Reply:
Fixed in 337fb20.